public-ui/src/app/data-browser/views/survey-view/survey-react-view.component.tsx

```diff
@@ -0,0 +1,451 @@
+    pdfLink: {
+        paddingTop: '1rem'
+    },
+    surveyResults: {
+        position: 'relative'
+    },
+    questionResult: {
+        borderTop: '1px solid #cccccc',
+        padding: '9px 0',
+        fontSize: '0.9em',
+        fontFamily: 'Arial, sans-serif',
+        color: '#262262',
+        width: '100%'
+    },
+    noResults: {
+        padding: '1em',
+        fontSize: '16px',
+        color: '#262262',
+        textAlign: 'left'
+    },
+    topicText: {
+        borderTop: '1px solid #cccccc',
+        fontSize: '1.2em',
+        fontWeight: 500,
+        paddingTop: '2%',
+        paddingBottom: '2%'
+    },
+    surveyView: {
+        boxSizing: 'border-box'
+    },
+    infoText: {
+        marginTop: '0',
+        color: '#262262'
+    }
+});
+
+const surveyStyle = `
+.toggle-link, .toggle-link:link, .toggle-link:visited {
+    color: #216fb4;
+}
+@media only screen and (max-width: 767px) {
+    .stat-container {
+        flex-basis: 100%;
+    }
+}
+.question-result:first-of-type .topic-text:first-of-type{
+    border-top: none;
+}
+.topic-text:first-of-type{
+    border-top: none;
+}
+.question .body-default {
+    padding-bottom: 18px;
+}
+@media only screen and (max-width: 768px) {
+    .survey-head {
+        flex-direction: column;
+    }
+    .cope-stat-layout {
+        width: 100%;
+        align-items: flex-end;
+        flex-direction: row;
+    }
+    .cope-stat-layout .stat-container {
+        padding-right: 0px;
+        padding-bottom: 2rem;
+    }
+    .version-table {
+        width: 100%;
+    }
+    .pdf-link {
+        position: absolute;
+        right: 0;
+        top: 0;
+        padding-top: 0;
+    }
+}
+.topic-text + .question-result {
+    border-top: none;
+}
+div {
+    display: block;
+}
+`;
+
+interface Props {
+    domainId: string;
+    searchWord: string;
+}
+
+interface State {
+    survey: any;
+    surveyPdfUrl: any;
+    isCopeSurvey: boolean;
+    searchWord: string;
+    extraQuestionConceptIds: Array<any>;
+    surveyVersions: Array<any>;
+    showAnswer: {};
+    questions: Array<any>;
+    loading: boolean;
+    showStatement: boolean;
+}
+
+export class SurveyViewReactComponent extends React.Component<Props, State> {
+    search = _.debounce(() => {
+          this.getSurvey();
+          this.fetchSurvey(this.props.domainId);
+        }, 1000);
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isCopeSurvey: false,
+            survey: null,
+            surveyPdfUrl: '',
+            searchWord: this.props.searchWord,
+            extraQuestionConceptIds: [],
+            showAnswer: {},
+            questions: [],
+            loading: true,
+            surveyVersions: [],
+            showStatement: false,
+        };
+        this.closePopUp = this.closePopUp.bind(this);
+    }
+
+    componentDidMount() {
+        this.fetchSurvey(this.props.domainId);
+    }
+
+    closePopUp() {
+        this.setState({showStatement: false});
```

Comment:
Sorry, missed this the first time around. Can move this inline in the template also and drop this function.